packages/schemas/src/sidebarDataLayoutSchema.ts

```diff
@@ -1,9 +1,16 @@
 import { z } from 'zod';
 
 export const sidebarDataLayoutSchema = z.union([
```

Comment:
Only the layouts that had a sidebar were added here...why would we need to add layouts that don't have a sidebar?

Reply:
Good shout... I had a sidebar and changed the layout and then saw the error and thought this schema validated layouts.. was on my radar to ask you, if folks add their own layouts, this would need to be extended.
I can put it back to what it was (my bad). Although, how do we see folks adding their own layouts that use sidebars ?

Reply:
restored previous change